src/phoenix/server/api/routers/v1/__init__.py

```diff
@@ -40,8 +40,10 @@ def __init__(self, path: str, endpoint: Callable[..., Any], **kwargs: Any) -> No
     Route("/v1/evaluations", evaluations.post_evaluations, methods=["POST"]),
     Route("/v1/evaluations", evaluations.get_evaluations, methods=["GET"]),
     Route("/v1/traces", traces.post_traces, methods=["POST"]),
+    Route("/v1/traces/annotate", traces.annotate_traces, methods=["POST"]),
```

Comment:
Avoid verbs in the route?

Reply:
`/v1/trace_annotations` for consistency with `/v1/experiment_evaluations`?

Reply:
that's a good point, will use this